ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -612,6 +612,7 @@ literals:
 - gas_volume_accumulator
 - water_volume_accumulator
 - gas_temperature_sensor
+- liquid_volume_accumulator
```

Comment:
I disagree with this use of the `liquid` subfield, if it is just a general flow meter, the `liquid` is unnecessary and `volume_accumulator` should be descriptive enough (see precedent with `flowrate_sensor`)

Reply:
Done the changes